src/framework/components/element/component.js

```diff
@@ -43,10 +84,51 @@ pc.extend(pc, function () {
             _warning = true;
         }
     };
+    
     ElementComponent = pc.inherits(ElementComponent, pc.Component);
 
-
     pc.extend(ElementComponent.prototype, {
+        
+        // Prepares stencil params for the inner components to be utilized during
+        // rendering. To keep things the least obtrusive way, it assumes the default stencil
+        // buffer value is 0, meaning the topmost mask (mind masking elements can be nested 
+        // into each other) should fill the buffer with a const using GREATEREQUAL function,
+        // while children should be drawn with smaller ref value and LESSEQUAL function.
+        _getStencilParameters: function() {
+            return new pc.StencilParameters({
```

Comment:
It's called "once" – upon actually chaiging the masking, the material or whatever else. However, agree re GC – will extract this to a field to be on the safe side.